src/utils/CompilerLocationUtils.fs

```diff
@@ -202,11 +202,14 @@ module internal FSharpEnvironment =
         // Check for an app.config setting to redirect the default compiler location
         // Like fsharp-compiler-location
         try 
+            // FSharp.Compiler support setting an appkey for compiler location. I've never seen this used.
```

Comment:
It's used [in this repo](https://github.com/Microsoft/visualfsharp/blob/fsharp4/vsintegration/src/unittests/Unittests.dll.config#L5)